tests/encodehelp.h

```diff
@@ -166,6 +172,13 @@ void ensureInputParameters()
     if (!fps)
         fps = 30;
 
+    if (ipbMode == 0)
```

Comment:
Aactually, my origial idea same as you, but finally I changed my mind because: 1. ipb_mode is more easy to know what it means. 2. ipb_mode is similar to avcenc's mode, it will be easier to compare yamienc and avcenc. 3. As our dirver and yami implemetaion, if user set a lager ipPerid, it will bring very obvious quality decline than IP mode, so ipPeriod eqaul to 3 is my recommend. User may refer our setting (IBBPBBP) after he read our test case. On the other hand, he can set ipPeriod to other values because we provide ipPeriod not ipbMode as yami interface.